src/Homework/Week3/Pb3/Message.java

```diff
@@ -0,0 +1,19 @@
+package Homework.Week3.Pb3;
+
+public class Message {
+    private String author;
```

Comment:
e ok, dar aici ar fi mers sa ai direct un field private User author si sa iei din User informatiile care te intereseaza cand printezi mesajul (daca cineva schimba numele unui User dupa momentul in care a fost creat mesajul, mesajul tau va avea asociat pentru totdeauna numele vechi)